repository/s.json

```diff
@@ -1,6 +1,17 @@
 {
 	"schema_version": "3.0.0",
 	"packages": [
+		{
+			"name": "Saklash-Theme",
+			"details": "https://github.com/SahilChandravanshi/Saklash-Theme-sublime",
+			"labels": ["color scheme"],
+			"releases": [
+				{
+					"sublime_text": "*",
+					"branch": "main"
```

Comment:
Branch based releases are no longer supported and you're already using tags. Change this to `"tags": true`.